LocalExtensions.php

```diff
@@ -194,6 +194,22 @@
 
 if ( $wmgUseCommentStreams ) {
 	wfLoadExtension ( 'CommentStreams' );
+	$wi->config->settings['wgManageWikiNamespacesAdditional']['default']['wgCommentStreamsAllowedNamespaces'] = [
+		'name' => 'Can comments appear in this namespace?',
+		'main' => true,
+		'talk' => true,
+		'blacklisted' => [],
+		'vestyle' => false,
+		'overridedefault' => -1,
```

Comment:
Should be null.

Reply:
I had based myself on  https://www.mediawiki.org/wiki/Topic:Uecwz5ba26qbnu0t